releasenotes/notes/fix-preference-window-header-color,-align-close-button-and-change-borders-to-margins-fa7ffffc45b12ea5.yaml

```diff
@@ -0,0 +1,29 @@
+fixes:
+  - |
+    fix preferences window header color, align the close button more nicely and change borders to margins
+
+translations:
+    - pt_BR
+    - sv
```

Comment:
i have reverted this part, these translations have not been updated (or if you speak 20 languages?), just mo files are generated. translations => something updated the translation

Reply:
No I actually changed all of them by hand (poedit) I removed some markup which was possible without knowing the language. You can remove it from the releasenote but the po files are changed by me (all of them)

Reply:
yes of course i keep the change in po. thanks for that !